### PR TITLE
Centralize developer logs

### PR DESCRIPTION
**Pull request by rtnpro**
Dump all developer logs to ``/srv/pipeline-logs/cccp.log``. This will help us to monitor cccp logs by tailing a single file: ``tail -f /srv/pipeline-logs/cccp.log``

### CODE REVIEW

**Comment by navidshaikh** [comment]
We need to make available volume to mail-server container as well.

**Comment by rtnpro** [comment]
@navidshaikh thanks for bring this up. I will add logging to the mail server container as well.

**Comment by navidshaikh** [comment]
PR #161 is refactoring the mail-server container entrypoint file, if we can get mentioned PR in first before this, will help reduce rebase issues.

File: provisions/roles/openshift/tasks/setup_mailservice.yml [hunk 1]
```diff
@@ -21,3 +25,5 @@
       state: started
```
Comment (navidshaikh) [inline provisions/roles/openshift/tasks/setup_mailservice.yml]: should we also change the state of this container as of others, like `restarted` ?

**Comment by navidshaikh** [comment]
> PR #161 is refactoring the mail-server container entrypoint file, if we can get mentioned PR in first before this, will help reduce rebase issues.

@bamachrn ping ?

**Comment by navidshaikh** [comment]
We merged the PR even before CI could run completely ?

`1 check was pending`

**Pull request merged**